### PR TITLE
Add missing rest api handler tests

### PR DESCRIPTION
**Pull request by pwirth**

### CODE REVIEW

File: tests/phpunit/integration/DAApiTest.php [hunk 1]
```diff
@@ -50,6 +55,125 @@ public function testGetServerInfo(): void {
 		$this->assertSame( ServerInfo::DA_API_VERSION, $data['api_version'] );
 	}
 
+	/**
+	 * @covers \DataAccounting\API\GetHashChainInfoHandler
+	 */
+	public function testHashChainInfoHandler(): void {
+		// Testing the case when the rev_id is found.
+		$services = [
```
Comment (rht) [inline tests/phpunit/integration/DAApiTest.php:63]: I'm merging, but also wondering if it is possible to deduplicate these similar steps when the rev_id is found, permissions denied, and testing expected fields in the json. Not a priority though.

Comment (rht) [reply tests/phpunit/integration/DAApiTest.php:63]: Made #253.

**Comment by rht** [comment]
The failures in the test are actual failures:
```
2) DataAccounting\Tests\DAApiTest::testGetRevisionHashesHandler
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-'Permission denied'
+'Not found'
```

**Comment by pwirth** [comment]
> The failures in the test are actual failures:
> 
> ```
> 2) DataAccounting\Tests\DAApiTest::testGetRevisionHashesHandler
> Failed asserting that two strings are identical.
> --- Expected
> +++ Actual
> @@ @@
> -'Permission denied'
> +'Not found'
> ```

failures in the code or failures in the test?

**Comment by rht** [comment]
I found that errs in the CI log.

**Comment by rht** [comment]
Probably need to rebase and force push.

**Pull request merged**